local/bin/py/build/configurations/pull_config_preview.yaml

```diff
@@ -321,13 +321,57 @@
     - action: pull-and-push-file
       branch: master
       globs:
-      - 'docs/rum_collection.md'
+      - 'docs/rum_getting_started'
       options:
-        dest_path: '/real_user_monitoring/'
+        dest_path: '/real_user_monitoring/android'
         file_name: 'android.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_collection.md"]
-          title: Getting Started with Android RUM Collection
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_getting_started.md"]
+          title: RUM Android Monitoring
+          kind: documentation
+    - action: pull-and-push-file
+      branch: master
+      globs:
+      - 'docs/mobile_data_collected.md'
+      options:
+        dest_path: '/real_user_monitoring/android/data_collected'
+        file_name: 'mobile_data_collected.md'
+        front_matters:
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/mobile_data_collected.md"]
+          title: RUM Android Data Collected
+          kind: documentation
+    - action: pull-and-push-file
+      branch: master
+      globs:
+      - 'docs/configure_rum_android_sdk.md'
+      options:
+        dest_path: '/real_user_monitoring/android/advanced_configuration'
+        file_name: 'configure_rum_android_sdk.md'
```

Comment:
```suggestion
        dest_path: '/real_user_monitoring/android/'
        file_name: 'advanced_configuration.md'
```

Reply:
This file is called configure_rum_sdk.md in its original path -> would that be a problem?

https://github.com/DataDog/dd-sdk-android/blob/c80fef149ea92d3a6f63aa1a7487ea3a62d0b66e/docs/configure_rum_android_sdk.md